Lib/test/test_dbm.py

```diff
@@ -129,6 +129,9 @@ def test_anydbm_access(self):
         assert(f[key] == b"Python:")
         f.close()
 
+    def test_open_with_pathlib_path(self):
```

Comment:
Add please a test for bytes path (`os.fsencode(_fname)`).

Reply:
I now have tests for four cases:

- `str`
- `FakePath` with a `str` value
- `bytes`
- `FakePath` with a `bytes` value

AFAIK, all valid input should fall into one of these situations.